clang/bindings/python/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools>=42", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "clang"
+description = "libclang python bindings"
+readme = {file = "README.txt", content-type = "text/plain"}
+
+license = { text = "Apache-2.0 with LLVM exception" }
+authors = [
+    { name = "LLVM" }
+]
+keywords = ["llvm", "clang", "libclang"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Development Status :: 5 - Production/Stable",
+    "Topic :: Software Development :: Compilers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+]
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "http://clang.llvm.org/"
+Download = "http://llvm.org/releases/download.html"
```

Comment:
https links please. Also consider adding relevant source, documentation, etc.

Reply:
Is there an official documentation site with info for the clang python bindings?

Reply:
I don't know about where the relevant documentation for that is, if any. Maybe wait for a developer to respond on this

Reply:
> Is there an official documentation site with info for the clang python bindings?

I don't think there is, but the most promising route seems to get https://libclang.readthedocs.io official and regularly updated.

Reply:
It looks like @sighingnow is the one who maintains that site (doc build scripts at https://github.com/sighingnow/libclang/tree/master/docs). If he's okay with it (my RTD username is rmast), I can get that documentation site updated.

Otherwise, the docs are just built with sphinx so we could upload it as a subpage to one of the documentation sites under the llvm.org domain -- if the source code for those sites is under the llvm org I can look at opening a PR to generate the libclang Python binding docs and deploying them to a website.